File: wireless/v1/rate_plan/create-example-1/create-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

rate_plan = @client.wireless
                   .rate_plans
                   .create(
                      friendly_name: 'California SmartMeter Plan',
                      messaging_enabled: true,
                      unique_name: 'SmartMeterCA'
                    )

puts rate_plan.sid

File: wireless/v1/rate_plan/create-example-1/create-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Wireless.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var ratePlan = RatePlanResource.Create(
            friendlyName: "California SmartMeter Plan",
            messagingEnabled: true,
            uniqueName: "SmartMeterCA"
        );

        Console.WriteLine(ratePlan.Sid);
    }
}
